lib/src/telegram/models/message.dart

```diff
@@ -19,7 +19,7 @@ part of '../model.dart';
 /// This object represents a message.
 ///
 /// https://core.telegram.org/bots/api#message
-@JsonSerializable()
+@JsonSerializable(explicitToJson: true)
```

Comment:
I believe this is not necessary since you have already set it in `build.yml`

Reply:
Yes, you're right.